tests/Test.hs

```diff
@@ -661,7 +661,6 @@ genCoreF subgen varGen =
         -- , CoreIdentEq _ <$> sameVars <*> sameVars
         -- , CoreSyntax Syntax
         -- , CoreCase sameVars [(Pattern, core)]
-        -- , CoreIdentifier Ident
         -- , CoreDataCase core [(ConstructorPattern, core)]
         -- , CoreIdent (ScopedIdent core)
```

Comment:
despite the name, `CoreIdent` is not the core representation of `ident`, it's the core representation of `ident-syntax`.